frontend/src/pages/PostRegister/PostRegister.tsx

```diff
@@ -264,7 +264,7 @@ const PostForm = (props: PostFormProps) => {
               <br />
               {/* // TODO: mess with button component to accept optional props} */}
               <Button
-                className='submit'
+                classNameS='submit'
```

Comment:
You sure about this?

Reply:
Yes. because we don't want to give a class name to the actual button. we want to give a class name to the span that wraps the button.